### PR TITLE
Updated README with npd fields and added npd metrics docs

### PR DESCRIPTION
**Pull request by Ab-hishek**
This PR adds metrics details added exported by node-problem-detector and also updates the README with the npd fields used in values.yaml.

### CODE REVIEW

**Review by shubham14bajpai — APPROVED** [review]
LGTM

File: docs/metrics-node-problem-detector.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# node-problem-detector metrics
+
```
Comment (avishnu) [inline docs/metrics-node-problem-detector.md:2]: problem*

Comment (Ab-hishek) [reply docs/metrics-node-problem-detector.md:2]: It is problem only Vishnu.

Comment (avishnu) [reply docs/metrics-node-problem-detector.md:2]: @Ab-hishek , its the below line.

Comment (Ab-hishek) [reply docs/metrics-node-problem-detector.md:2]: Ohkay.

**Review by avishnu — APPROVED** [review]
lgtm

**Pull request merged**